README.md

```diff
@@ -94,6 +94,7 @@ jobs:
 | `build`     | build the dist file. You are required to run this locally in order to build the dist before opening a PR.                                                                                                                                       |
 | `run:local:report` | Run the action on two local folders, for testing and development. The first argument is the old ("base branch") folder and the sedcond argument is the new ("head branch") folder. For example `yarn run:local:report ./test-data/old ./test-data/new` |
 | `run:local:debugfile` | Show what the program parses from a single file, after stripping all non-wanted items from the file, and before rating the readability. For example `yarn run:local:debugfile ./test-data/new/test-document.md` |
+| `playground:dev` | Open the playground in local development mode for testing readability scores |
```

Comment:
I think the decrease in readability is because this sentence doesn't end with a period

Reply:
I added periods to them all and the score didn't change.

Reply:
I'm not sure what decreased the score then 😕

Reply:
Its a small amount of content so I think the sore can be sway easily by the addition or removal of a few lines.